datastore/src/main/java/io/spine/server/storage/datastore/delivery/SessionRecordColumn.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2021, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.storage.datastore.delivery;
+
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.Timestamp;
+import io.spine.query.RecordColumn;
+import io.spine.query.RecordColumns;
+import io.spine.server.delivery.ShardSessionRecord;
+
+import static io.spine.query.RecordColumn.create;
+
+/**
+ * The definitions of record columns to store along with the {@link ShardSessionRecord}.
+ *
+ * @apiNote This type is made {@code public} to allow library users query the stored
+ *         {@code ShardSessionRecord}s via storage API, in case they need to read
+ *         the storage contents manually.
```

Comment:
Please also document on the reason for violating the Java standard for naming fields.

Reply:
This is documented by `@RecordColumns` annotation by which this type is marked. We discussed that previously, and introduced it specifically for this reason.

However, as you have comments on this, we'll probably need to discuss this one more time — which is fine with me. I will think this over.